### DISCUSSION TITLE
Add last activity to the hover popup in the "Recent activity sidebar"

### DISCUSSION THREAD

**Discussion by escamoteur** [community/community#123732] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

If I hover over a last recent activity entry it only show me something like this
![grafik](https://github.com/community/community/assets/16635729/c83b50cb-0607-457b-836c-4f67e8fab5a4)
and it will only state if I have commented or mentioned but not what the last activity was. 
So I don't know if someone else has commented after me e.g. so I have to to visit the issues or PR to see if something has really changed which is pretty annoying if you have a lot of activities. 
Adding the date of the last activity like

You commented on 1. April 2024
@username commented on 1. April 

would greatly enhance this view.
Also highlighting any entry that has changed since I last visited would be helpful.

**Comment by Wid1ey** [comment]
I am not exactly sure what functionality you are referring to.. my apologies

**Reply by Wid1ey** [reply]
how to locate relevant information at each step of setting up Feedly to track GitHub updates:

Sign Up or Log In to Feedly:

Action: Visit the Feedly website (https://feedly.com/) and locate the sign-up or log-in section on the homepage.
Relevant Information: Enter your desired username, email address, and chosen password into the respective fields. If you already have an account, simply log in with your existing credentials.
Add GitHub RSS Feed:

Action: After logging in, you'll see a search bar at the top of the Feedly interface. Click on it to expand the search functionality.
Relevant Information: Enter the URL of the GitHub repository you want to track into the search bar and press Enter. This will display the available feeds related to the repository.
Follow the Feed:

Action: Once the search results appear, click on the feed that corresponds to the type of updates you want to track (e.g., Commits, Releases).
Relevant Information: No additional information needed at this step. Simply click on the "Follow" button next to the feed to add it to your Feedly account.
Organize Feeds into Collections (Optional):

Action: After following the desired feeds, you can organize them into collections for easier management.
Relevant Information: Click on the "Collections" tab on the left sidebar to create a new collection or select an existing one. Give your collection a descriptive name (e.g., GitHub Updates) and drag the relevant feeds into it.
Configure Notification Settings:

Action: Once your feeds are organized, you can configure notification settings for each feed or collection.
Relevant Information: Click on the bell icon (Notification Settings) next to the feed or collection you want to configure. Choose your preferred notification frequency (e.g., Immediate, Daily Digest) and delivery method (e.g., Email, Browser Notification).
Explore Additional Features:

Action: Take some time to explore Feedly's additional features, such as keyword alerts and integrations with other productivity tools.
Relevant Information: Navigate through the Feedly interface and look for options related to advanced customization, keyword alerts, and integrations. These features can enhance your tracking experience based on your specific needs.
Enjoy Tracking GitHub Updates:

Action: Once you've completed the setup process, you can start enjoying tracking GitHub updates through Feedly.
Relevant Information: Visit your Feedly dashboard to access and manage the tracked GitHub updates. You can view them in your collections and adjust settings as needed.
By following these steps and locating the relevant information as outlined, you'll be able to set up Feedly to effectively track GitHub updates according to your specific requirements. If you have any questions or encounter any difficulties along the way, feel free to ask for assistance!

**Reply by LiteBrite82** [reply]
Hi @Wid1ey , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.
